### PR TITLE
update "Publish Content"

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
Based on demo video: https://salesforce.quip.com/05alAxnQ9Whc

### CODE REVIEW

File: modules/ROOT/pages/publish-content.adoc [hunk 1]
```diff
@@ -4,33 +4,71 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-With ACM you can create and publish content articles including news, tutorials, developer blog posts, and events.
+With ACM you can create, organize, and publish content articles including news, tutorials, developer blog posts, and events.
 
-Select the *Manage Communities* tab and click *Content Management*.
+== Create a new article
```
Comment (valkyrie69) [inline modules/ROOT/pages/publish-content.adoc]: Title case for titles

File: modules/ROOT/pages/publish-content.adoc [hunk 2]
```diff
@@ -4,33 +4,71 @@ include::_attributes.adoc[]
...
-The list shows all of your community's articles.
+The Salesforce Community Cloud *Content Management* interface appears and shows a list of your community's articles.
 
-Click the *Create Content* button in the top right.
+Select the *Salesforce CMS* button at the top right.
 
-image::create-content.png[Create Content page]
+The *Salesforce CMS* interface shows the workspaces in your Salesforce organization. These workspaces contain the content articles.
+
+The *Salesforce CMS* interface enables a team such as marketing or business development to own the production and publication of content articles in a single place, and re-use the content across multiple communities.
```
Comment (valkyrie69) [inline modules/ROOT/pages/publish-content.adoc]: reuse (no hyphen)

File: modules/ROOT/pages/publish-content.adoc [hunk 3]
```diff
@@ -4,33 +4,71 @@ include::_attributes.adoc[]
...
+
+Click the name of the workspace where you would like to create a new article.
```
Comment (valkyrie69) [inline modules/ROOT/pages/publish-content.adoc]: where you want to (instead of would like)

File: modules/ROOT/pages/publish-content.adoc [hunk 4]
```diff
@@ -4,33 +4,71 @@ include::_attributes.adoc[]
...
 Fill in the article's *Title*, *Body*, *Featured Image*, and *Excerpt*. The *Excerpt* is the preview text on an article's carousel tile.
 
-Set the article's *Topics*. Topics are case-sensitive. As you start to type, a drop-down list shows existing topics you can choose.
+Click the *Save Draft* button at the top right.
+
+You can make more edits and save the draft again any number of times. The article will not be presented in any collection because it is not published.
+
+When you are ready, click the *Publish* button at the top right.
+
+== Add the article to a collection
```
Comment (valkyrie69) [inline modules/ROOT/pages/publish-content.adoc]: Title case

**Review by valkyrie69 — APPROVED** [review]
very minor comments. No show stoppers.

**Comment by wwarner-salesforce** [comment]
Thank you! Corrections are made.

**Pull request merged**